devutils/dbdatagen/data/aepoints.json

```diff
@@ -1,34 +1,42 @@
 {
-  "first": {
-    "description" : "FIRST Program Workshops, Presentations, and Socials",
-    "value" : 3
+  "career" : {
+    "description" : "University Career Center Testing & Events",
+    "value" : 1
   },
-  "professor": {
-    "description" : "Professor Meetings",
-    "value" : 4
+  "community" : {
+    "description" : "Community Involvement Activities",
+    "value" : 1
   },
-  "staff": {
-    "description" : "Advisor/Faculty/Staff Meetings",
+  "firstother" : {
+    "description" : "FIRST Program Presentations & Socials",
     "value" : 1
   },
-  "outreach": {
-    "description" : "FIRST Program Outreach",
-    "value" : 2
+  "firstworkshops" : {
+    "description" : "FIRST Program Workshops",
+    "value" : 3
   },
-  "community": {
-    "description" : "Community Involvement Activities",
+  "mentor" : {
+    "description" : "Mentor Meetings",
     "value" : 1
   },
-  "other": {
+  "other" : {
     "description" : "Other College of Engineering Workshops & Engineering Company Info Sessions",
     "value" : 1
   },
-  "mentor": {
-    "description" : "Mentor Meetings",
+  "outreach" : {
+    "description" : "FIRST Program Outreach",
     "value" : 2
   },
-  "career": {
-    "description" : "University Career Center Testing & Events",
+  "professor" : {
+    "description" : "Professor Meetings",
+    "value" : 4
+  },
+  "staff" : {
+    "description" : "Advisor/Faculty/Staff Meetings",
     "value" : 1
+  },
+  "other": {
```

Comment:
When I first came up with the names, I used the "other" key for "Other College of Engineering Workshops & Engineering Company Info Sessions", so we'll have to change one of the keys. Probably the original one. Maybe something like "otherCOE"?